<?xml version="1.0" encoding="UTF-8"?>
<testsuite hostname="hadoop04" name="com.takako.MoneyTest" tests="13" failures="1" timestamp="29 Sep 2011 23:54:09 GMT" time="0.031" errors="0">
  <testcase name="testCurrency" time="0.0" classname="com.takako.MoneyTest"/>
  <testcase name="testEquality" time="0.0" classname="com.takako.MoneyTest"/>
  <testcase name="testIdentityRate" time="0.0" classname="com.takako.MoneyTest"/>
  <testcase name="testMixedAddition" time="0.0" classname="com.takako.MoneyTest"/>
  <testcase name="testMultiplication" time="0.0" classname="com.takako.MoneyTest"/>
  <testcase name="testPlusReturnsSum" time="0.0" classname="com.takako.MoneyTest"/>
  <testcase name="testPlusSameCurrencyReturnMoney" time="0.0" classname="com.takako.MoneyTest">
    <failure type="java.lang.AssertionError">
      <![CDATA[java.lang.AssertionError: 
	at org.testng.AssertJUnit.fail(AssertJUnit.java:59)
	at org.testng.AssertJUnit.assertTrue(AssertJUnit.java:24)
	at org.testng.AssertJUnit.assertTrue(AssertJUnit.java:33)
	at com.takako.MoneyTest.testPlusSameCurrencyReturnMoney(MoneyTest.java:114)
	at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
	at sun.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:39)
	at sun.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:25)
	at java.lang.reflect.Method.invoke(Method.java:597)
	at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:80)
	at org.testng.internal.Invoker.invokeMethod(Invoker.java:676)
	at org.testng.internal.Invoker.invokeTestMethod(Invoker.java:845)
	at org.testng.internal.Invoker.invokeTestMethods(Invoker.java:1169)
	at org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:125)
	at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:109)
	at org.testng.TestRunner.runWorkers(TestRunner.java:1182)
	at org.testng.TestRunner.privateRun(TestRunner.java:761)
	at org.testng.TestRunner.run(TestRunner.java:612)
	at org.testng.SuiteRunner.runTest(SuiteRunner.java:335)
	at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:330)
	at org.testng.SuiteRunner.privateRun(SuiteRunner.java:292)
	at org.testng.SuiteRunner.run(SuiteRunner.java:241)
	at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:52)
	at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:86)
	at org.testng.TestNG.runSuitesSequentially(TestNG.java:1169)
	at org.testng.TestNG.runSuitesLocally(TestNG.java:1094)
	at org.testng.TestNG.run(TestNG.java:1006)
	at org.testng.remote.RemoteTestNG.run(RemoteTestNG.java:107)
	at org.testng.remote.RemoteTestNG.initAndRun(RemoteTestNG.java:199)
	at org.testng.remote.RemoteTestNG.main(RemoteTestNG.java:170)
]]>
    </failure>
  </testcase>
  <testcase name="testReduceMoney" time="0.0" classname="com.takako.MoneyTest"/>
  <testcase name="testReduceMoneyDifferentCurrency" time="0.0" classname="com.takako.MoneyTest"/>
  <testcase name="testReduceSum" time="0.0" classname="com.takako.MoneyTest"/>
  <testcase name="testSimpleAddition" time="0.0" classname="com.takako.MoneyTest"/>
  <testcase name="testSumPlusMoney" time="0.0" classname="com.takako.MoneyTest"/>
  <testcase name="testSumTimes" time="0.0" classname="com.takako.MoneyTest"/>
</testsuite>
